options/options.go

```diff
@@ -0,0 +1,46 @@
+package options
+
+import (
+	"github.com/cockroachdb/pebble"
+	ldb "github.com/syndtr/goleveldb/leveldb/opt"
+)
+
+//Contains all available read/write options
+//for the supported databases. Fields are set
+//by functions using the SetOptions signature.
+type Options struct {
+	LeveldbRead  *ldb.ReadOptions
+	LeveldbWrite *ldb.WriteOptions
+	PebbleWrite  *pebble.WriteOptions
```

Comment:
Note that because these fields are pointers - we're allocating only an int64 pointer for each option in the struct, so we don't have a lot of overhead defining all options in the `Options` struct.